Cleanup teams doc, particularly around slack moderation (#1524)
writethedocs/www: docs/team.rst

you can see who is helping out:

* :doc:`Portland 2021 Team </conf/portland/2021/team>`
* :doc:`Prague 2021 Team </conf/prague/2021/team>`
* :doc:`Australia 2021 Team </conf/australia/2021/team>`

.. _moderation-team:

Moderation Team
---------------

> This team is charged with keeping our Slack network on topic and

Reviewer: Charged with feels a little heavy, but might just be me

```suggestion
This team helps keep our Slack network on topic and
```
